tests/test_corpus.py

```diff
@@ -125,8 +125,9 @@ def test_ud():
 
 def test_open_url():
     corpus = teanga.from_url("https://teanga.io/examples/ex1.yaml")
-    assert len(corpus.docs) == 1
-    assert corpus.docs[0][1].text.text[0] == "Teanga2 data model"
+    corpus_docs = list(corpus.docs)
```

Comment:
I'm guessing this caused trouble before I got to reviewing the files because `corpus` is now a dict object.